source/MRMesh/MRSurfaceDistance.h

```diff
@@ -39,6 +39,9 @@ MRMESH_API VertScalars computeSurfaceDistances( const Mesh& mesh, const MeshTriP
 MRMESH_API VertScalars computeSurfaceDistances( const Mesh& mesh, const MeshTriPoint & start, float maxDist = FLT_MAX,
                                                          const VertBitSet* region = nullptr, int maxVertUpdates = 3 );
 
+MRMESH_API VertScalars computeSurfaceDistances( const Mesh& mesh, const std::vector<MeshTriPoint>& start, float maxDist = FLT_MAX,
```

Comment:
Please add a comment to the new function

Reply:
And `start` -> `starts`

Reply:
fixed